tests/fs-tree/fs-test.js

```diff
@@ -1402,34 +1409,57 @@ describe('FSTree fs abstraction', function() {
       });
 
       tree.writeFileSync('omg.js', 'hi');
+      tree.writeFileSync('hello.txt', "Hello Again, World!\n");
+      tree.writeFileSync('my-directory/goodbye.txt', "Goodbye, World!\n");
     })
 
     afterEach(function() {
       tree.unlinkSync('omg.js');
+      tree.unlinkSync('my-directory/goodbye.txt');
     })
 
     it('hides no changes if all match', function() {
-      let changes = tree.changes({ include: ['**/*.js']});
+      let filter = { include: ['**/*'] };
+      let changes = tree.filtered(filter).changes();
 
-      expect(changes).to.have.property('length', 1);
+      expect(changes).to.have.property('length', 3);
       expect(changes).to.have.deep.property('0.length', 3);
       expect(changes).to.have.deep.property('0.0', 'create');
       expect(changes).to.have.deep.property('0.1', 'omg.js');
+      expect(changes).to.have.deep.property('1.length', 3);
+      expect(changes).to.have.deep.property('1.0', 'change');
+      expect(changes).to.have.deep.property('1.1', 'hello.txt');
+      expect(changes).to.have.deep.property('2.length', 3);
+      expect(changes).to.have.deep.property('2.0', 'create');
+      expect(changes).to.have.deep.property('2.1', 'my-directory/goodbye.txt');
     });
 
-
     it('hides changes if none match', function() {
-      expect(tree.changes({ include: ['NO-MATCH'] })).to.have.property('length', 0);
+      expect(tree.filtered({ include: ['NO_MATCH'] }).changes()).to.have.property('length', 0);
     });
 
     it('hides changes if they are outside of cwd', function() {
-      expect('this thing is tested').to.equal(true);
+      let changes = tree.chdir('my-directory').changes();
+
+      expect(changes).to.have.property('length', 1);
+      expect(changes).to.have.deep.property('0.length', 3);
+      expect(changes).to.have.deep.property('0.0', 'create');
+      expect(changes).to.have.deep.property('0.1', 'my-directory/goodbye.txt');
```

Comment:
The second arg to changes should be `cwd`-aware.

Also it might be worthwhile to add assertions about the third arg, ie the `entry`